generator/src/test/java/org/apache/kafka/message/VersionConditionalTest.java

```diff
@@ -37,7 +35,7 @@ static void assertEquals(CodeBuffer buffer, String... lines) throws Exception {
         for (String line : lines) {
             expectedStringBuilder.append(String.format(line));
         }
-        Assert.assertEquals(expectedStringBuilder.toString(), stringWriter.toString());
+        Assertions.assertEquals(stringWriter.toString(), expectedStringBuilder.toString());
```

Comment:
We don't use the prefix ```Assertions``` now.

Reply:
I tried to remove the prefix Assertions.